tests/test_wave_02.py

```diff
@@ -55,3 +55,42 @@ def test_get_tasks_sorted_desc(client, three_tasks):
             "is_complete": False,
             "title": "Answer forgotten email 📧"},
     ]
+
+
+# NEW UNIT TESTS
+def test_get_tasks_sort_empty(client, three_tasks):
+    response = client.get("/tasks?sort=")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert len(response_body) == 3
+
+def test_get_tasks_sort_case_insensitive(client, three_tasks):
+    response = client.get("/tasks?sort=DESC")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == [
+        {
+            "description": "",
+            "id": 1,
+            "is_complete": False,
+            "title": "Water the garden 🌷"},
+        {
+            "description": "",
+            "id": 3,
+            "is_complete": False,
+            "title": "Pay my outstanding tickets 😭"},
+        {
+            "description": "",
+            "id": 2,
+            "is_complete": False,
+            "title": "Answer forgotten email 📧"},
+    ]
+
+def test_get_tasks_sort_query_invalid(client, three_tasks):
```

Comment:
Nice added tests again! Totally cool to have interpreted the invalid sort query string param as necessitating sending back a 400 but note that if you adopted my strategy of returning an empty list, this test would need to be modified!